src/test/java/com/thealgorithms/strings/IsomorphicTest.java

```diff
@@ -1,31 +1,21 @@
 package com.thealgorithms.strings;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.jupiter.api.Test;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 public final class IsomorphicTest {
-    private IsomorphicTest() {
-    }
-
-    @Test
-    public static void main(String[] args) {
-        String str1 = "abbbbaac";
-        String str2 = "kffffkkd";
-
-        String str3 = "xyxyxy";
-        String str4 = "bnbnbn";
 
-        String str5 = "ghjknnmm";
-        String str6 = "wertpopo";
-
-        String str7 = "aaammmnnn";
-        String str8 = "ggghhhbbj";
+    @ParameterizedTest
+    @MethodSource("isomorphicProvider")
+    public void testCheckStrings(String str1, String str2, Boolean expected) {
+        assertEquals(expected, Isomorphic.checkStrings(str1, str2));
```

Comment:
What do you think about:
```suggestion
        assertEquals(expected, Isomorphic.checkStrings(str1, str2));
        assertEquals(expected, Isomorphic.checkStrings(str2, str1));
```